packages/kbn-types/package.json

```diff
@@ -3,11 +3,12 @@
   "version": "0.0.1",
   "types": "./types/packages/kbn-types/index.d.ts",
   "scripts": {
-    "prepare": "npm run build",
     "build": "tsc",
     "start": "tsc --watch"
   },
   "devDependencies": {
+    "@elastic/kbn-observable": "link:../kbn-observable",
+    "@elastic/kbn-utils": "link:../kbn-utils",
```

Comment:
This is just because we can't depend directly on the platform itself, right now. Maybe we need to re-work that at some point, but this is likely okey for now.